mods/ascent/machines/magnetotron.dm

```diff
@@ -0,0 +1,74 @@
+// Mutation device for turning alates into gynes.
+/obj/machinery/ascent_magnetotron
+	name = "magnetotron"
+	desc = "An assemblage of alien machinery with large imposing dynamos."
+	color = COLOR_PURPLE
+	icon = 'icons/obj/stationobjs.dmi'
+	icon_state = "weightlifter"
+
+/obj/machinery/ascent_magnetotron/attack_hand(var/mob/user)
+	var/mob/living/carbon/human/target
+	for(var/mob/living/carbon/human/T in loc)
+		target = T
+		break
+
+	if(isnull(target))
+		to_chat(user, SPAN_WARNING("\icon[src] \The [src] chirps, \"No biological signature detected in [src].\""))
```

Comment:
Purely a lore nitpick, but Ascent machinery would not use vocal warning messages, and people unfamiliar with Voc or Glow wouldn't understand it.